### ISSUE TITLE
link wrong document

### ISSUE THREAD

**Issue by aureliosaraiva** [ember-cli-deploy/ember-cli-deploy#426]
During the testing process ember-cli-deploy I was notified deprecated feature, but the link that shows how to solve does not work.

> Use of the `config.plugins` property has been deprecated. Please use the new plugin run controls.
> See the following page for information:
> http://ember-cli-deploy.com/docs/v1.0.x-beta.1/configuration/

link correct: http://ember-cli-deploy.com/docs/v1.0.0-beta.1/configuration/#advanced-plugin-configuration

**Comment by ghedamat** [comment]
agreed @aureliosaraiva that got missed when we changed the documentation path for the beta release

probably the easiest thing to do would be to add a redirect to the `gh-pages` branch of this repo. should be doable with jekyll but we'll have to investigate..

**Comment by lukemelia** [comment]
@ghedamat does this still need action?

**Comment by objectiveSee** [comment]
@lukemelia I just received this error today.

```
> ember --version

ember-cli: 2.10.0
node: 5.9.1
os: darwin x64
```

**Comment by ghedamat** [comment]
@lukemelia looks like I missed that redirect or something else is not configured properly. I'll try have a look this week

**Comment by ghedamat** [comment]
should be solved in #461 when merged and new version of ember-cli-deploy is released
